quizzes/midterm.md

```diff
@@ -142,13 +196,35 @@
 	  other_hash = { new_key: "new value", new_another: "I will overwrite or will I?"}
 	  my_hash.merge(other_hash)
 	```
-
+	
+	```
+	my_hash.merge(other_hash) = { "another value I will overwrite or will I?" }
+	```
 
 1. What is the first thing you should do with `Legacy Code?`. And no, you can't throw it out.
+	```
+	The first thing that you should do with Legacy Code is to test it.
+	```
 
 1. Who is the creator of Ruby?
+	```
+	Matz or Yukihiro something
+	````
 
 1. Write a class that uses a Global, Instance, Class, and local variable. Don't worry about what it does, just properly use the variables.
+	```
+	class Frustration
+	   $mood = mood
+	   def initialize(first_name, last)
+	      @first_name = first_name
+	      @last = last
+	      @@feeling = feeling
+	      
+	      puts "#{first_name} #{last} is #{@@feeling} #{$mood}.
+	   end
+	end
+	      
+	```
 
 1. Write a Monkey class that `eat`s bananas, and `drink`s water
```

Comment:
10/10 - you didn't need to overwrite the animal classes drink method. But I'll give it to you because it works.